Logic/igtlioConnector.cxx

```diff
@@ -341,209 +310,220 @@ void igtlioConnector::RequestPushOutgoingMessages()
 }
 
 //----------------------------------------------------------------------------
-int igtlioConnector::WaitForConnection()
+void* igtlioConnector::ConnectionThreadFunction(void* ptr)
```

Comment:
"ConnectionThreadFunction" is not very specific. Something like "ConnectionAcceptThreadFunction" would be more descriptive.